examples/with-webassembly/pages/api/edge.js

```diff
@@ -0,0 +1,11 @@
+import add_module from '../../add.wasm?module'
```

Comment:
Should we have a standalone example for WASM usage in the Edge runtime? I think this one is mostly for WASM in Next.js pages.